client/header/activity-panel/panels/inbox/card.js

```diff
@@ -26,21 +31,79 @@ class InboxNoteCard extends Component {
 		this.state = {
 			isDismissModalOpen: false,
 			dismissType: null,
+			screen: this.getScreenName(),
```

Comment:
This is working well, but does it need to be part of state? Can it just be an instance variable?

Reply:
Good idea @jeffstieler, I removed the `screen` from the state in the commit [4ac394b](https://github.com/woocommerce/woocommerce-admin/pull/4320/commits/4ac394b7213218ac1c48691f61a6b150924d1907)